aten/src/ATen/SparseTensorUtils.h

```diff
@@ -2,36 +2,38 @@
 
 #include <ATen/ATen.h>
 #include <ATen/SparseTensorImpl.h>
+#include <ATen/Parallel.h>
 
 namespace at { namespace sparse {
 
 // Just for documentary purposes
 using SparseTensor = Tensor;
-using LongTensor = Tensor;
 using IntTensor = Tensor;
+using LongTensor = Tensor;
```

Comment:
We should kill these and just use Tensor for clarity and consistency with the rest of PyTorch's code style. It doesn't have to be in this PR.